my_view_page.php

```diff
@@ -151,7 +151,13 @@
 
 <?php if( $t_timeline_view_threshold_access ) { ?>
 <div class="col-md-5 col-xs-12">
-	<?php include( $g_core_path . 'timeline_inc.php' ); ?>
+	<?php
+		# Build a simple filter that gets all bugs for current project
+		$inc_timeline_filter = array();
+		$inc_timeline_filter[FILTER_PROPERTY_HIDE_STATUS] = array( META_FILTER_NONE );
+		$inc_timeline_filter = filter_ensure_valid_filter( $inc_timeline_filter );
```

Comment:
Is it better to ensure valid here or in the API?  What pattern should be using and why?